internal/distro/fedora/distro.go

```diff
@@ -680,10 +680,11 @@ func (t *imageType) initializeManifest(bp *blueprint.Blueprint,
 	repos []rpmmd.RepoConfig,
 	packageSets map[string]rpmmd.PackageSet,
 	containers []container.Spec,
-	seed int64) (*manifest.Manifest, error) {
+	seed int64) (*manifest.Manifest, []string, error) {
 
-	if err := t.checkOptions(bp.Customizations, options, containers); err != nil {
-		return nil, err
+	warnings, err := t.checkOptions(bp.Customizations, options, containers)
+	if err != nil {
+		return nil, warnings, err
```

Comment:
I'm not sure if it makes sense to return warnings if there is an error... 🤔  This applies to all places where we check for an error.

I'm thinking whether we should return warnings only if we return `nil` as the error from this function.